ports/openfx/Openfx.cmake

```
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.20)
+
+project(openfx VERSION 1.4.0 LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+if(WIN32)
+    add_compile_definitions(WINDOWS)
+    add_compile_options(/DNOMINMAX)
+    add_definitions(-DWIN64)
```

Three ways to achieve similar effects?
```suggestion
    add_compile_definitions(WINDOWS NOMINMAX WIN64)
```